test/extended/setup.sh

```diff
@@ -2,6 +2,8 @@
 #
 # This abstracts starting up an extended server.
 
+source "${OS_ROOT}/hack/util.sh"
```

Comment:
We should `source` `hack/lib/init.sh` in the script that's calling into this one, which will pick up whatever is in `util.sh` as well.